Makefile

```diff
@@ -3,10 +3,17 @@ TARGET = alacritty
 APP_NAME = Alacritty.app
 ASSETS_DIR = extra
 RELEASE_DIR = target/release
+COMPLETIONS_DIR = $(ASSETS_DIR)/completions
+
+EXTRAS = $(ASSETS_DIR)/alacritty.info $(ASSETS_DIR)/alacritty.man
+COMPLETIONS = $(COMPLETIONS_DIR)/_alacritty $(COMPLETIONS_DIR)/alacritty.bash $(COMPLETIONS_DIR)/alacritty.fish
+
```

Comment:
If we want to use whitespace to separate things logically, we should have some reason to it. I currently don't really see why we would add a newline before and after extra/completions.

We can probably move `APP_NAME` just before `APP_TEMPLATE` and then combine `ASSETS_DIR`, `RELEASE_DIR`, `COMPLETIONS_DIR`, `EXTRAS` and `COMPLETIONS` into one block.